app/controllers/wishlists_controller.rb

```diff
@@ -0,0 +1,22 @@
+class WishlistsController < ApplicationController
+  def new
+    @wishlist = Wishlist.new
+  end
+
+  def create
+    @wishlist = Wishlist.new(wishlist_params)
+    @wishlist.user_id = current_user.id
```

Comment:
Consider consolidating this line with the one above by:

1. Adding a `has_many :wishlists` association on User
2. Using the following:

```ruby
@wishlist = current_user.wishlists.new(wishlist_params)
```